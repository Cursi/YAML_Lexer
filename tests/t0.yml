---
# values
a: 5.23 #muhaha
b: 'string value' # testaaaaaawadawdwad
c: value
d: '123'