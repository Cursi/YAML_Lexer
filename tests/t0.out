| document: 
| | map value
| | | key: [a]
| | | numeric value: [5]
| | | key: [b]
| | | string value: [string value]
| | | key: [c]
| | | string value: [value]
| | | key: [d]
| | | string value: [125]
